### PR TITLE
feat: filled in readme and added actions

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #1 
Added readme and actions files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<a href="http://artipie.com"><img src="https://www.artipie.com/logo.svg" width="64px" height="64px"/></a>
+
+[![Join our Telegramm group](https://img.shields.io/badge/Join%20us-Telegram-blue?&logo=telegram&?link=http://right&link=http://t.me/artipie)](http://t.me/artipie)
+
+[![EO principles respected here](https://www.elegantobjects.org/badge.svg)](https://www.elegantobjects.org)
+[![DevOps By Rultor.com](http://www.rultor.com/b/artipie/cocoa-adapter)](http://www.rultor.com/p/artipie/cocoa-adapter)
```
Comment (dgarus) [inline README.md]: We don't use rultor.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+[![We recommend IntelliJ IDEA](https://www.elegantobjects.org/intellij-idea.svg)](https://www.jetbrains.com/idea/)
+
+[![Javadoc](http://www.javadoc.io/badge/com.artipie/cocoa-adapter.svg)](http://www.javadoc.io/doc/com.artipie/cocoa-adapter)
+[![License](https://img.shields.io/badge/license-MIT-green.svg)](https://github.com/com.artipie/cocoa-adapter/blob/master/LICENSE.txt)
+[![codecov](https://codecov.io/gh/artipie/cocoa-adapter/branch/master/graph/badge.svg)](https://codecov.io/gh/artipie/cocoa-adapter)
+[![Hits-of-Code](https://hitsofcode.com/github/artipie/cocoa-adapter)](https://hitsofcode.com/view/github/artipie/cocoa-adapter)
+[![Maven Central](https://img.shields.io/maven-central/v/com.artipie/cocoa-adapter.svg)](https://maven-badges.herokuapp.com/maven-central/com.artipie/cocoa-adapter)
+[![PDD status](http://www.0pdd.com/svg?name=artipie/cocoa-adapter)](http://www.0pdd.com/p?name=artipie/cocoa-adapter)
```
Comment (dgarus) [inline README.md]: Suggest do not use PDD, just create issues on GitHub.

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
Minor comments.

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected

**Review by dgarus — APPROVED** [review]
@olenagerasimova thanks! LGTM

**Pull request merged**